### ISSUE TITLE
Update RepositoryCard styling to match ExtensionCard

### ISSUE THREAD

**Issue by msangapu-msft** [craigshoemaker/livewire#32]

**Comment by craigshoemaker** [comment]
When I went to implement the logic for the VS Code extensions, I realized that I needed to pull the same data from GitHub for an extension in the same way for any repo. The end result is that the UI components can now be unified to just be a `ResourceCard` rather than having individual `ExtensionCard` and `RepositoryCard` components. 

Basically we can just rename `RepositoryCard` to `ResourceCard` and then we'll be done 😉 

Closed by 6ef1965a2cad819b22f55ed458b4300f50f47ef3

**Comment by msangapu-msft** [comment]
@craigshoemaker I was wondering what happened to this. I was scratching my head and then I went through the issues to find this. I'm glad you were able to simplify it.

Can you tell me how this works? Where is .test function? Curious to see how the conditional rendering is happening.

![image](https://user-images.githubusercontent.com/49697512/95388456-f2c58380-08b7-11eb-9b07-7cafc0f16fdf.png)

**Comment by craigshoemaker** [comment]
@msangapu-msft  The [`test`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) method is available off an instance of [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp). Using this approach makes it easy to evaluate for the different values possible for `PartitionKey` in a clean way (rather than a series of `===` expressions).

Basically you pass the `test` method a string and it evaluates to a `bool` based on whether or not the  string matches your given regex pattern.

**Comment by msangapu-msft** [comment]
Got it... I understood it was testing somehow, didn't know it was RegEx. Very cool.

I really like how you unified the cards. It's very simple. One of the reasons I felt we needed separate cards was due to the different HTML output. Today I looked closer and it renders the same HTML regardless of the card type:

![image](https://user-images.githubusercontent.com/49697512/95474066-53050580-094a-11eb-8312-e7eb2bfa1086.png)


I haven't thought about "how" yet, but I was hoping there was a way to check the card type within this component so we can exclude the unnecessary HTML for extension cards. It's a minor thing, so feel free to veto it, but I'd still like to learn how to do it in vue.js even if we don't implement it! 

Either way, we should definitely keep this simple with one card instead of two. Awesome work, Craig! I'm learning a lot!
